src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -46,6 +46,11 @@ public boolean isMaintainer() {
     return this.isMaintainer;
   }
 
+  //TODO(): get correct moderator status for organization
+  public boolean isModerator(String organization) {
```

Comment:
Would organization represent organization's ID?

Reply:
yes, will modify to say organizationID